frame/nomination-pools/src/lib.rs

```diff
@@ -538,19 +538,80 @@ pub struct PoolRoles<AccountId> {
 	pub state_toggler: Option<AccountId>,
 }
 
+/// Pool commission.
+///
+/// The pool depositor can set a commission upon a pool creation, from which the pool owner can
+/// update thereafter. The `max` commission value can only be set once, as to prevent the commission
+/// from repeatedly increasing.
+///
+/// A commission throttle is also optional, allowing the pool to set strict limits to how much
+/// commission can change in each update, and how often updates can take place.
+#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, DebugNoBound, PartialEq, Copy, Clone)]
+#[codec(mel_bound(T: Config))]
+#[scale_info(skip_type_params(T))]
+pub struct Commission<T: Config> {
+	/// The active commission rate of the pool.
+	current: Perbill,
+	/// An optional maximum commission that can be set by the pool root. Once set, this value
+	/// cannot be updated.
+	max: Option<Perbill>,
+	/// The account commission is paid to.
+	receiver: Option<T::AccountId>,
```

Comment:
Beneficiary?

Reply:
Also, this could be grouped with current in an option. If current is set (even if zero, although that's trivial to prevent) the beneficiary must exist. If not set, no need to have a beneficiary either. We would also then have one transaction to set this, not two.

Reply:
> Beneficiary?

`payee`? Keeping things consistent with nominating.